source/reference/limitations.txt

```diff
@@ -54,7 +54,8 @@ General Limitations
 - .. include:: /includes/fact-applyOps.rst
 - ``mongosync`` must read from the source cluster using the 
   :readmode:`primary` read preference.
-
+- ``mongosync`` does not work on source or destination clusters that 
```

Comment:
could we instead say "source or destination clusters that are currently upgrading MongoDB versions are not supported"? We use "not supported" instead of "does not work" for our other limitation bullets

Reply:
Sure thing, just updated.